src/styles/components/blog-card.scss

```diff
@@ -1,49 +1,47 @@
-@use "../base/mixins";
+@use "../base/mixins" as *;
 .blog-card {
   display: flex;
   flex-direction: column;
   gap: 16px;
 
   &__article-writer {
-    @include mixins.text-style(
+    @include text-style(
       var(--card-text),
       var(--primary-font),
       var(--font-small),
       600
     );
-    @include mixins.spacing(20px, left);
+    @include spacing(20px, left);
     order: -1;
   }
 
   &__date {
-    @include mixins.text-style(
+    @include text-style(
       var(--card-text),
       var(--primary-font),
       var(--font-small),
       600
     );
```

Comment:
I think you can try to add it to parent, because font styles is inherited
<img width="400" alt="Снимок экрана 2025-05-12 в 7 00 43 PM" src="https://github.com/user-attachments/assets/906e416a-1f02-43ad-ac5a-347587d76c61" />